module/documents/token.mjs

```diff
@@ -29,6 +29,7 @@ export default class TokenDocument5e extends SystemFlagsMixin(TokenDocument) {
    * @type {string}
    */
   get subjectPath() {
+    if ( game.release.generation >= 12 ) return this.ring.subject.texture;
```

Comment:
Is this full path guaranteed to exist? Or do we need some optional chaining?

Reply:
Guaranteed to exist.